src/translator/service.cpp

```diff
@@ -6,33 +6,48 @@
 #include <utility>
 
 inline std::vector<marian::Ptr<const marian::Vocab>>
-loadVocabularies(marian::Ptr<marian::Options> options) {
+loadVocabularies(marian::Ptr<marian::Options> options, const std::vector<marian::bergamot::AlignedMemory>& vocabMemorySet) {
   // @TODO: parallelize vocab loading for faster startup
-  auto vfiles = options->get<std::vector<std::string>>("vocabs");
-  // with the current setup, we need at least two vocabs: src and trg
-  ABORT_IF(vfiles.size() < 2, "Insufficient number of vocabularies.");
```

Comment:
Do we have this check somewhere in the aligned memory case? Should be useful, I hope?